jwt/api.py

```diff
@@ -16,7 +16,7 @@
 
 
 class PyJWT(object):
-    def __init__(self, algorithms=None):
+    def __init__(self, algorithms=None, options=None):
```

Comment:
There's no reason why we couldn't use `options=` for other options later. I feel like `verification_options` would restrict what we could do with this argument unnecessarily.

In addition, all the current options start with `verify_` so I feel like it'd be a bit redundant to call it `verification_options`.